src/CodeGenerator/.github/workflows/ci.yml

```diff
@@ -0,0 +1,37 @@
+name: Tests
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
```

Comment:
If the project is a sub-tree split, what's the purpose of this file (especially the `pull_request` option

Reply:
`pull_request`  has no meaning. But we will get nice badges in the readme =)